ads/google/a4a/traffic-experiments.js

```diff
@@ -90,8 +91,8 @@ export function googleAdsIsA4AEnabled(win, element, experimentName,
   const isSetFromUrl = maybeSetExperimentFromUrl(win, element,
       experimentName, externalBranches.control,
       externalBranches.experiment, delayedExternalBranches.control,
-      delayedExternalBranches.experiment,
-      MANUAL_EXPERIMENT_ID);
+      delayedExternalBranches.experiment, sfgInternalBranches.control,
+      sfgInternalBranches.experiment, MANUAL_EXPERIMENT_ID);
   const experimentInfoMap = {};
   const branches = [
     internalBranches.control,
```

Comment:
...actually, based on your code comment on line 210, I am probably wrong about that.